src/Annotation/ApiResource.php

```diff
@@ -251,19 +270,38 @@ final class ApiResource
     /**
      * @see https://github.com/Haehnchen/idea-php-annotation-plugin/issues/112
      *
+     * @var string|false
+     */
+    private $input;
+
+    /**
+     * @see https://github.com/Haehnchen/idea-php-annotation-plugin/issues/112
+     *
+     * @var int
+     *
+     * @deprecated - Use $paginationMaximumItemsPerPage instead
+     */
+    private $maximumItemsPerPage;
+
+    /**
+     * @see https://github.com/Haehnchen/idea-php-annotation-plugin/issues/112
+     * @see https://api-platform.com/docs/core/mercure/#pushing-the-api-updates
+     *
      * @var mixed
      */
     private $mercure;
 
     /**
      * @see https://github.com/Haehnchen/idea-php-annotation-plugin/issues/112
+     * @see https://api-platform.com/docs/core/messenger/#using-messenger-with-an-input-object
```

Comment:
Should be: https://api-platform.com/docs/core/messenger/#dispatching-a-resource-through-the-message-bus